src/Services/network.js

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+
+const client = axios.create({
+    baseURL: ""
+});
+
+
+export const fetchData = async (url) => {
+    try {
+        const response = await client.get(url);
+        return response.data;
+    } catch (error) {
+        return {}
```

Comment:
In case of an error, it should return the error message based on the backend response, not {}